packages/graphile-build-pg/src/plugins/PgTypesPlugin.js

```diff
@@ -660,32 +621,17 @@ export default (function PgTypesPlugin(
         }
         gqlTypeByTypeIdAndModifier[type.id][typeModifierKey] = Range;
         gqlInputTypeByTypeIdAndModifier[type.id][typeModifierKey] = RangeInput;
+        if (pgTweaksByTypeIdAndModifer[type.id] === undefined) {
+          pgTweaksByTypeIdAndModifer[type.id] = {};
+        }
+        pgTweaksByTypeIdAndModifer[type.id][
+          typeModifierKey
+        ] = fragment => sql.fragment`case when (${fragment}) is null then null else json_build_object(
+          'start', json_build_object('value', lower(${fragment}), 'inclusive', lower_inc(${fragment})),
+          'end', json_build_object('value', upper(${fragment}), 'inclusive', upper_inc(${fragment}))
```

Comment:
To avoid the breaking schema changes this PR introduces, can we instead set the start/end null in SQL:

```sql
          'start', case when lower(${fragment}) is null then null else json_build_object('value', lower(${fragment}), 'inclusive', lower_inc(${fragment})) end,
          'end', case when upper(${fragment}) is null then null else json_build_object('value', upper(${fragment}), 'inclusive', upper_inc(${fragment})) end
```